content/lessons/chapter-4/address-3.tsx

```diff
@@ -71,14 +76,16 @@ ${
     hiddenRange: [1, 0, 160, 0],
     constraints: [
       {
-        range: [166, 1, 166, 1],
+        range: [166, 1, 168, 1],
         allowMultiline: true,
       },
     ],
   }
 
   const python = {
-    program: `print("KILL")`,
+    program: `
```

Comment:
We need to update the solution in `address.tsx` in resources to reflect this change.